src/SQLQueryStress/LoadEngine.cs

```diff
@@ -558,46 +558,5 @@ public class QueryOutput
             }
              */
         }
-

```

Comment:
Why was this removed?

Reply:
And looks like very old code..

Reply:
It looks like it's not used by any of other classes, removed for "Avoid uninstantiated internal classes"